### PR TITLE
adds documentation on multisite setup

### LINKED ISSUE #1208: Document Drupal multisite use case
This has been brought up a few times recently, so it will be good to have it documented on [docs.docksal.io](https://docs.docksal.io/)

Example of a D7 multisite setup:
https://github.com/docksal/boilerplate-drupal7-advanced

Relevant docs:

- https://docs.docksal.io/fin/fin-help/#db
- https://docs.docksal.io/tools/drupal/
- https://docs.docksal.io/core/system-vhost-proxy/#using-arbitrary-custom-domains

### PR DESCRIPTION
**Pull request by shelane**
Close #1208

### CODE REVIEW

File: docs/content/use-cases/drupal-multisite.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: "Drupal Multisite"
+weight: 1
+---
+
+>Drupal has a feature which allows separate, independent sites to be served from a single codebase. Each site has its own 
+database, configuration, files and base domain or URL.
```
Comment (achekulaev) [inline docs/content/use-cases/drupal-multisite.md]:
```suggestion
database, configuration, public files folder and base domain or URL.
```

File: docs/content/use-cases/drupal-multisite.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+A `default` database is created with every project that uses the `default` stack, but each multisite requires its own database.
+Therefore, you will need to create the database in the container for each multisite that is not the default site.
+
+```bash
+fin db create 'anothersite'
+```
+
+### Create a Site Directory for the New Site
+
+Each site has its own directory with a settings.php file. You will have the settings for your live site in there and you
```
Comment (achekulaev) [inline docs/content/use-cases/drupal-multisite.md]:
```suggestion
Each site has its own directory under `docroot/sites` containing a `settings.php` file, in which you will have the settings for your live site and you 
```

File: docs/content/use-cases/drupal-multisite.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Therefore, you will need to create the database in the container for each multisite that is not the default site.
+
+```bash
+fin db create 'anothersite'
+```
+
+### Create a Site Directory for the New Site
+
+Each site has its own directory with a settings.php file. You will have the settings for your live site in there and you 
+should have a `local.settings.php` file where you include local settings. Your `settings.php` file should detect and include
```
Comment (achekulaev) [inline docs/content/use-cases/drupal-multisite.md]:
```suggestion
should have a `local.settings.php` file in which you include settings for the local copy of your site. Your `settings.php` file should detect and include `local.settings.php`.
```

File: docs/content/use-cases/drupal-multisite.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+the local file.
```
Comment (achekulaev) [inline docs/content/use-cases/drupal-multisite.md]:
```suggestion
```

File: docs/content/use-cases/drupal-multisite.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+      'driver' => 'mysql',
+      'prefix' => '',
+    ],
+  ],
+];
+```
+
+### Add to sites.php
+
+For Drupal to direct a domain to a defined multisite, it must be defined in the `sites.php` file. Add the following line
```
Comment (achekulaev) [inline docs/content/use-cases/drupal-multisite.md]:
```suggestion
Sub-site domain must be defined in the `sites.php` file for Drupal to redirect its URLs to it. Add the following line
```

File: docs/content/use-cases/drupal-multisite.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+to your `sites.php` file and replace `myproject` with your Docksal project name.
```
Comment (achekulaev) [inline docs/content/use-cases/drupal-multisite.md]:
```suggestion
to your `sites.php` file, and replace `myproject` with your Docksal project name.
```

File: docs/content/use-cases/drupal-multisite.md (hunk 1 above)
Comment (shelane) [reply docs/content/use-cases/drupal-multisite.md]: Hmmm. As I start tinkering with words, I'm feeling a small rewrite here is in order.

File: docs/content/use-cases/drupal-multisite.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+```bash
+fin db create 'anothersite'
+```
+
+## Create a Site Directory for the New Site
+
+Each site has its own directory under `docroot/sites` that should contain a `settings.php` file where you have the settings 
+for your live site. You should also have a `local.settings.php` file that includes settings for the local copy of your
+site. Your `settings.php` file should detect and include that file.
```
Comment (achekulaev) [inline docs/content/use-cases/drupal-multisite.md]:
```suggestion
site. `settings.php` should detect and include `local.settings.php`.
```

File: docs/content/use-cases/drupal-multisite.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+Sub-site domains must be defined in the `sites.php` file for Drupal to redirect its URLs to it. Add the following line
+to your `sites.php` file, and replace `myproject` with your Docksal project name.
+
+```php
+$sites['anothersite.myproject.docksal'] = 'anothersite';
+```
+Your site will then be accessible through `http://anothersite.myproject.docksal` (substitute your project URL).
+
+You can look at the []Drupal 7 advanced boilerplate project](https://github.com/docksal/boilerplate-drupal7-advanced) to see an example.
```
Comment (achekulaev) [inline docs/content/use-cases/drupal-multisite.md]:
```suggestion
You can look at the [Drupal 7 advanced boilerplate project](https://github.com/docksal/boilerplate-drupal7-advanced) to see an example.
```

**Pull request merged**